### PR TITLE
feat: remove old variables

### PR DESCRIPTION
**Pull request by droshev**
remove old variables

removed classes: `fd-app`, `fd-ui`, `fd-ui--fixed`, `fd-ui__header`, `fd-ui__footer`, `fd-ui__app`, `fd-ui__main`, `fd-ui__overlay`, `fd-has-background-image`, `fd-has-background-fixed`, `fd-has-background-size-contain`, `fd-has-background-size-cover`, `fd-has-background-color-transparent`, `fd-has-background-color-none`, `fd-has-background-color-*`, `fd-has-border-radius-*`, `fd-has-border-*`, `fd-has-border-color-*`, `fd-has-box-shadow-none`, `fd-has-color-*`, `fd-display*`, `fd-has-text-transform-*`, `fd-has-font-weight-*`, `fd-has-font-style-*`, `fd-has-text-align-*`, `fd-has-float-*`, `fd-has-height-*`, `fd-has-margin-*`, `fd-has-padding-*`, `fd-has-type-*`, `fd-has-font-*`, `sap-icon--s`, `sap-icon--small`, `sap-icon--m`, `sap-icon--medium`, `sap-icon--l`, `sap-icon--large`, `sap-icon--xl`, `sap-icon--xlarge`, `sap-icon--animate-spin`, `sap-icon--animate-pulse`, `fd-nav+(--vertical)`, `fd-nav__item`, `.fd-nav__link`, `fd-product-tile`, `fd-product-tile__media`, `fd-product-tile__title`, `fd-product-tile__text`

removed variables: `--fd-color-*`, `--fd-background-color*`, `--fd-forms-*`, `--fd-padding*`, `--fd-width-*`, `--fd-section-*`

fixes: #812

### CODE REVIEW

File: src/icons/_settings.scss [hunk 1]
```diff
@@ -1,9 +1,6 @@
-@import "../settings";
-@import "../functions";
-
 // sizes
 $fd-icons-sizes: (
-  "default": $fd-font-size, // 14px
+  "default": 14px,
   "s": 12px,
   "m": 16px,
   "l": 20px,
```
Comment (InnaAtanasova) [inline src/icons/_settings.scss:6]: maybe these should be in rem?

File: src/tree.scss [hunk 2]
```diff
@@ -14,46 +12,32 @@ $block: #{$fd-namespace}-tree;
...
+  $fd-tree-link-color: #0a6ed1 !default;
 
-  --fd-tree-cell-spacing: var(--fd-width-gutter);
-  $fd-tree-cell-padding: fd-space(3) !default;
-  $fd-tree-header-color: fd-color("text", 3) !default;
-  $fd-tree-header-background-color: fd-color("neutral", 2) !default;
-  $fd-tree-row-background-color--hover: fd-color-state("hover") !default;
+  $fd-tree-header-color: #6a6d70 !default;
+  $fd-tree-header-background-color: #eeeeef !default;
+  $fd-tree-row-background-color--hover: #fafafa !default;
```
Comment (InnaAtanasova) [inline src/tree.scss:22]: why not css variables from sap-theming?

File: src/tree.scss [hunk 3]
```diff
@@ -14,46 +12,32 @@ $block: #{$fd-namespace}-tree;
...
 
   $fd-tree-control-width: 18px !default;
   $fd-tree-control-button-width-height: 34px !default;
```
Comment (InnaAtanasova) [inline src/tree.scss:25]: in rem?

File: src/tree.scss [hunk 4]
```diff
@@ -14,46 +12,32 @@ $block: #{$fd-namespace}-tree;
...
   border-bottom-style: solid;
   border-bottom-width: $fd-tree-border-width;
-
-  @include fd-var-color("border-bottom-color", $fd-tree-border-color, --fd-tree-border-color);
-
+  border-bottom-color: $fd-tree-border-color;
   padding-left: 0;
   padding-right: 0;
-  margin-bottom: $fd-margin-bottom;
+  margin-bottom: 12px;
```
Comment (InnaAtanasova) [inline src/tree.scss:40]: in rem?

File: src/tree.scss [hunk 5]
```diff
@@ -93,18 +74,16 @@ $block: #{$fd-namespace}-tree;
...
     list-style: none;
   }
 
   &__row {
     @include fd-reset();
-    @include fd-var-size("padding-left", $fd-tree-cell-spacing, --fd-tree-cell-spacing);
-    @include fd-var-size("padding-right", $fd-tree-cell-spacing, --fd-tree-cell-spacing);
 
+    padding-left: 16px;
+    padding-right: 16px;
```
Comment (InnaAtanasova) [inline src/tree.scss:86]: rem?

File: src/tree.scss [hunk 6]
```diff
@@ -128,24 +106,21 @@ $block: #{$fd-namespace}-tree;
...
     }
   }
-  $_row_padding: $fd-tree-control-width + $fd-tree-control-margin-right + 4;
 
   &__col {
     @include fd-reset();
 
     flex: 1;
-    padding: $fd-tree-cell-padding 0;
+    padding: 12px 0;
```
Comment (InnaAtanasova) [inline src/tree.scss:117]: rem

File: src/tree.scss [hunk 7]
```diff
@@ -128,24 +106,21 @@ $block: #{$fd-namespace}-tree;
...
     // pad to account for control
-    padding-left: $_row_padding;
+    padding-left: 30px;
```
Comment (InnaAtanasova) [inline src/tree.scss:119]: you can create a local variable and reuse it

File: src/tree.scss [hunk 8]
```diff
@@ -190,10 +165,10 @@ $block: #{$fd-namespace}-tree;
     height: $fd-tree-control-button-width-height;
     transform: rotate(180deg);
     vertical-align: middle;
-    transition: transform $fd-animation-speed linear;
+    transition: transform 0.125s linear;
+    color: #0a6ed1;
```
Comment (InnaAtanasova) [inline src/tree.scss:169]: hard-coded color

File: src/section.scss [hunk 9]
```diff
@@ -50,27 +49,36 @@ $block: #{$fd-namespace}-section;
...
+
+      @include fd-screen(m) {
+        padding-left: 2rem;
+        padding-right: 2rem;
+      }
+
+      @include fd-screen(xl) {
+        padding-left: 3rem;
+        padding-right: 3rem;
+      }
```
Comment (InnaAtanasova) [inline src/section.scss:63]: all these values are used twice in the file, maybe create variables for them

File: src/mixins/_icons.scss [hunk 10]
```diff
@@ -19,14 +19,13 @@
     font-style: normal;
     font-weight: normal;
     text-align: center;
-    font-size: $fd-font-size;
+    font-size: 0.875rem;
```
Comment (InnaAtanasova) [inline src/mixins/_icons.scss:22]: there's a variable from sap-theming

File: src/_new-settings.scss [hunk 11]
```diff
@@ -4,6 +4,27 @@
 // library namespace
 $fd-namespace: fd !default;
 
+$fd-breakpoints: (
+  s: 320px,    // phone     <600
+  m: 600px,    // tablet    600-1023
+  l: 1024px,   // desktop   1024-1439
+  xl: 1440px,  // desktop   1440-1919
+  xxl: 1920px, // display   >1920
+) !default;
```
Comment (InnaAtanasova) [inline src/_new-settings.scss:13]: If you are bringing this maybe consider using fiori3 breakpoints:
Screen Shot 2020-07-06 at 11 05 12 AM

File: src/icons/_settings.scss (hunk 1 above)
Comment (droshev) [reply src/icons/_settings.scss:6]: I am working on that and I will push soon

File: src/tree.scss (hunk 2 above)
Comment (droshev) [reply src/tree.scss:22]: we will scarp this soon, won't we ?

File: src/tree.scss (hunk 3 above)
Comment (droshev) [reply src/tree.scss:25]: I left it as is as we are refactoring it

File: src/helpers/_borders.scss [hunk 12]
```diff
@@ -1,4 +1,4 @@
-@import "./../settings";
+@import "./../new-settings";
 @import "../mixins/mixins";
 
 .#{$fd-namespace}-has-border-radius-0 {
```
Comment (droshev) [inline src/helpers/_borders.scss:4]: delete those

File: src/helpers/_box-shadow.scss [hunk 13]
```diff
@@ -1,4 +1,4 @@
-@import "./../settings";
+@import "./../new-settings";
 
 .#{$fd-namespace}-has-box-shadow-none {
```
Comment (droshev) [inline src/helpers/_box-shadow.scss:3]: delete and refactor

File: src/icons/_settings.scss (hunk 1 above)
Comment (droshev) [reply src/icons/_settings.scss:6]: deleted

File: src/helpers/_box-shadow.scss (hunk 13 above)
Comment (droshev) [reply src/helpers/_box-shadow.scss:3]: deleted

File: src/helpers/_borders.scss (hunk 12 above)
Comment (droshev) [reply src/helpers/_borders.scss:4]: deleted

File: src/_new-settings.scss (hunk 11 above)
Comment (droshev) [reply src/_new-settings.scss:13]: i will refactor them in another PR as this one is big enough

**Pull request merged**